assets/package.json

```diff
@@ -73,54 +67,32 @@
 		"use-memo-one": "^1.1.2"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.14.8",
-		"@babel/plugin-transform-runtime": "^7.10.4",
```

Comment:
Why are these removed?

Reply:
Not needed here. `@babel/core` is installed automatically as a dependency for other packages that we use, `@babel/plugin-transform-runtime` is not used in our setup.